stdlib/public/core/Algorithm.swift

```diff
@@ -119,17 +121,19 @@ public struct EnumeratedSequence<Base : Sequence> : Sequence {
 public struct EnumerateGenerator<Base : IteratorProtocol> { }
 
 @available(*, unavailable, renamed: "EnumeratedSequence")
-public struct EnumerateSequence<Base : Sequence> {}
+public struct EnumerateSequence<Base : Sequence> { }
 
 extension EnumeratedIterator {
-  @available(*, unavailable, message: "use the 'enumerated()' method on the sequence")
+  // Calling `makeIterator()` on an `EnumeratedSequence` is currently
+  // the only public way to obtain an `EnumeratedIterator`.
+  @available(*, unavailable, message: "Call 'enumerated().makeIterator()' on the sequence.")
   public init(_ base: Base) {
     Builtin.unreachable()
   }
 }
 
 extension EnumeratedSequence {
-  @available(*, unavailable, message: "use the 'enumerated()' method on the sequence")
+  @available(*, unavailable, message: "Use the 'enumerated()' method on the sequence.")
```

Comment:
This is tested for in `test/1_stdlib/Renames.swift`.  Could you update that test please?